changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## NEXT
+
+- Fixed a bug in `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.get_minimal_graph` where the method
+  was not adding the floating nodes correctly. This also impacted the graph extension.
```

Comment:
Have not had a chance to review full PR, but is there also a bug in `extend_graph` given our Slack discussion? Or does this fix also correct that?
```
ts = TimeSeriesCausalGraph()
ts.add_node('tier_0_0 lag(n=1)')
Out[13]: TimeSeriesNode("tier_0_0 lag(n=1)")
ts.extend_graph(1,0)
Out[14]: 
TimeSeriesCausalGraph(num_nodes=0, num_edges=0, id=-7856212696600437783)
Nodes: []
Edges: []
```

Reply:
It was due to just the minimal graph. I am double checking with the other PR though to make sure everything is consistent.